elements/lisk-p2p/src/p2p.ts

```diff
@@ -1008,6 +1022,8 @@ export class P2P extends EventEmitter {
 
 		// We need this check this._isActive in case the P2P library is shut down while it was in the middle of starting up.
 		if (this._isActive) {
+			// Initial Discovery SeedPeers and Disconnect
```

Comment:
According to LIP, we need to check if the number of previous peers is less than 100 then we do the discovery with seed peers. Otherwise, we choose only previous (tried) peers to do the discovery initially.